README.md

```diff
@@ -183,7 +183,10 @@ Example:
 - Twitter: [@twitterhandle](https://twitter.com/geekelo_xyz)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/eloghene-otiede)
 
+👤 **Mahdi Noori**
 
+- GitHub: [@Mahdi Noori](https://github.com/mahdinoori2000)
+- LinkedIn: [@Mahdi Noori](https://www.linkedin.com/in/mahdi-noori-hc201/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] The MIT license link in your readme leads to a 404 error page, ill suggest you add a license file and link it correctly as broken links make readme look a bit unprofessional :heavy_check_mark: :+1: 

- - Screenshot:
![Screenshot from 2023-09-05 10-54-17](https://github.com/geekelo/vet_clinic_database/assets/107632780/554cf9d0-4677-475c-b0bf-d38c9318a051)


- [ ] Also I'll suggest you go through your readme and remove these placeholder comments as they were added to guide you through and you were meant to remove them while customizing the readme. :smile: :small_airplane: 

- - Screenshot:
 
![Selection_016](https://github.com/geekelo/vet_clinic_database/assets/107632780/a09c66d6-9098-47ec-a221-4345fd86620a)